cms/static/sass/elements/_modal-window.scss

```diff
@@ -378,6 +383,9 @@
     }
   }
 
+  .modal-type-problem {
```

Comment:
this can move to modal-editor to apply to Video, HTML, Problems, and other Block Types. The JS that then determines the top / left pixel sizing might need a modification as a result of this, I'm not sure. This way the modal change applies to all blocks.